Library/Homebrew/test/emoji_spec.rb

```diff
@@ -4,8 +4,15 @@
   describe "#install_badge" do
     subject { described_class.install_badge }
 
-    it "returns 🍺 by default" do
-      expect(subject).to eq "🍺"
+    context "with emoji environment variables unset" do
```

Comment:
Agree the context isn't necessary.

@MikeMcQuaid there is a test for `HOMEBREW_INSTALL_BADGE` already, but not `HOMEBREW_NO_EMOJI`, I think because this method shouldn't even be called when that's set.